components/webfield/ReviewerConsole.js

```diff
@@ -367,6 +368,163 @@ const ReviewerConsoleTabs = ({
+ * @default no default value
+ * @example <caption>string shows single rating</caption>
+ * { "reviewRatingName": "rating" }
+ * @example <caption>string array shows multiple ratings</caption>
+ * { "reviewRatingName": ["soundness","excitement","reproducibility"] }
+ * @example <caption>object array/mixed shows multiple ratings with fallback options the following config would show 2 ratings: "overall_rating" and "overall_recommendation" for "overall_rating", it's value will be final_rating field, when final_rating field is not available, it will take the next available value defined in the array, in this example it will take "preliminary_rating"</caption>
+ * {
+ *  "reviewRatingName": [
+ *    {
+ *      "overall_rating": [
+ *        "final_rating",
+ *        "preliminary_rating"
+ *      ]
+ *    },
+ *    "overall_recommendation",
+ *  ]
+ */
+
+/**
+ * @name ReviewerConsoleConfig.areaChairName
+ * @description Used to construct AC/Anonymous AC group and label. optional for venues that don't have area chairs
+ * @type {string}
+ * @default no default value
+ * @example
+ * { "areaChairName": "Area_Chairs" }
+ */
+
+/**
+ * @name ReviewerConsoleConfig.submissionName
+ * @description Used to filter/construct group id (paper display/tasks), invitation id, header text
+ * @type {string}
+ * @default no default value
+ * @example
+ * { "submissionName": "Submission" }
+ */
+
+/**
+ * @name ReviewerConsoleConfig.submissionInvitationId
+ * @description Notes with the submissionInvitationId will be fetched
+ * @type {string}
+ * @default no default value
+ * @example
+ * { "submissionInvitationId": "ICLR.cc/202X/Conference/-/Submission" }
+ */
+
+/**
+ * @name ReviewerConsoleConfig.recruitmentInvitationId
+ * @description Related to customMaxPapersInvitationId and reviewLoad. The invitation to get recruitment note where custom load is saved (if there's no custom load edge) and custom load is displayed in header
+ * @type {string}
+ * @default no default value
+ * @example
+ * { "recruitmentInvitationId": "ICLR.cc/202X/Conference/Reviewers/-/Recruitment" }
+ */
+
+/**
+ * @name ReviewerConsoleConfig.customMaxPapersInvitationId
+ * @description Related to recruitmentInvitationId and reviewLoad. The invitation to get custom load edge. If edge exist the weight is used as custom load otherwise it will load the recruitment note to read the reduced_load field.
+ * @type {string}
+ * @default no default value
+ * @example
+ * { "customMaxPapersInvitationId": "ICLR.cc/202X/Conference/Reviewers/-/Custom_Max_Papers" }
+ */
+
+/**
+ * @name ReviewerConsoleConfig.reviewLoad
+ * @description Related to recruitmentInvitationId and customMaxPapersInvitationId. The default value to display in header when there's no custom load edge or recruitment note
+ * @type {string|number}
+ * @default no default value
+ * @example
+ * { "reviewLoad": "ICLR.cc/202X/Conference/Reviewers/-/Custom_Max_Papers" }
+ */
+
+/**
+ * @name ReviewerConsoleConfig.hasPaperRanking
```

Comment:
I don't think we support paper ranking in the workflow anymore so please remove it